x-pack/heartbeat/ironbank/Makefile

```diff
@@ -0,0 +1,46 @@
+BEAT_VERSION ?= 8.5.0
+LOCATION := ../build/heartbeat-ironbank-$(BEAT_VERSION)-docker-build-context
+
+YQ_VERSION ?= v4.13.2
+YQ ?= "../build/yq"
+OS = $(shell uname)
+ifeq ($(OS),Darwin)
+	YQ_BINARY ?= yq_darwin_amd64
+else
+	YQ_BINARY ?= yq_linux_amd64
+endif
+
+## @help:setup-yq: Install yq in ../build/yq.
+.PHONY: setup-yq
+setup-yq:
+	@if [ ! -f "../build/yq" ]; then \
+		echo "Downloading yq - $(YQ_VERSION)/$(YQ_BINARY)" ; \
+		curl -sSfL -o $(YQ) https://github.com/mikefarah/yq/releases/download/$(YQ_VERSION)/$(YQ_BINARY) ; \
+		chmod +x $(YQ) ; \
+	fi
```

Comment:
Avoid dependencies with `yq` in the workers or locally. It only supports `intel` so far. If needed, people could add support for `arm`